Source/BugsnagClient.m

```diff
@@ -25,6 +25,7 @@
 //
 
 #import "BugsnagClient.h"
+#import "BugsnagClientInternal.h"
```

Comment:
Do you think this is an approach we should take with all our public classes? To provide an internal header rather than all the class extensions we have dotted around?

Reply:
Ultimately yes, this would be a good idea. There are quite a few existing class extensions lying around so I think for now it'd be best to follow this pattern only if a class has a lot of properties/methods to hide. I'll create an internal ticket so that we can address this fully at a future date as it's a non-breaking change.